tests/phpunit/tests/Story_Renderer/HTML.php

```diff
@@ -226,6 +229,33 @@ public function test_sanitizes_and_optimizes_markup() {
 		$this->assertContains( 'AMP optimization could not be completed', $actual );
 	}
 
+	/**
+	 * @covers ::replace_url_scheme
+	 */
+	public function test_replace_url_scheme() {
+		$link = get_home_url( null, 'web-storires/test' );
+
+		$story    = new Story();
+		$renderer = new \Google\Web_Stories\Story_Renderer\HTML( $story );
+
+		$result = $this->call_private_method( $renderer, 'replace_url_scheme', [ set_url_scheme( $link, 'https' ) ] );
+		$this->assertEquals( $result, $link );
+	}
+
+
+	/**
+	 * @covers ::replace_url_scheme
+	 */
+	public function test_replace_url_scheme_invalid_url() {
```

Comment:
`invalid` doesn't feel like the best term here.

Perhaps something more verbose like this:

```suggestion
	public function test_replace_url_scheme_does_not_change_urls_with_different_host() {
```

or just

```suggestion
	public function test_replace_url_scheme_different_host() {
```